src/components/layout/GenericLayout/NavTools/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { MEDIA } from 'const'
+import { NavStyled } from '../Menu'
+import { Frame } from 'components/common/Frame'
+
+import SettingsImage from 'assets/img/settings.svg'
+import SettingsImageWhite from 'assets/img/settings-white.svg'
+import NotificationsImage from 'assets/img/bell.svg'
+import NotificationsImageWhite from 'assets/img/bell-white.svg'
+
+const SettingsLink = styled.li`
+  margin: 0 2.4rem 0 0;
+  order: 3;
+
+  > a {
+    display: block;
+    margin: 0;
+    width: 3.6rem;
+    height: 1.4rem;
+    content: '';
+    background: url(${SettingsImage}) no-repeat center/contain;
+  }
+
+  &:hover > a {
+    background: url(${SettingsImageWhite}) no-repeat center/contain;
+  }
+`
+
+const NotificationsLink = styled.li`
+  order: 4;
+
+  > a {
+    display: block;
+    margin: 0;
+    width: 3.6rem;
+    height: 1.4rem;
+    content: '';
+    background: url(${NotificationsImage}) no-repeat center/contain;
+  }
+
+  &:hover > a {
+    background: url(${NotificationsImageWhite}) no-repeat center/contain;
+  }
+`
+
+const Wrapper = styled(NavStyled)`
+  margin: 0 0 0 auto;
+  padding: 0;
+
+  @media ${MEDIA.mediumDown} {
+    flex-direction: row;
+    justify-content: flex-start;
+    justify-self: center;
+    padding: 1rem;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    z-index: 99;
+    height: 7.2rem;
+    border-radius: 0;
+    background-color: var(--color-gradient-2);
+  }
+
+  > li {
+    display: flex;
+    align-items: center;
+
+    &:not(:last-of-type)::after {
+      content: '';
+      display: block;
+      width: 0.1rem;
+      height: 2.4rem;
+      background: var(--color-border);
+      margin: 0 1.6rem;
+      position: absolute;
+      right: -2.8rem;
+    }
```

Comment:
Is this for pipe | separator? This is quite a lot of artificial code. Maybe better have a `<Separator/>` component used inside NavTools
Plus it probably shouldn't be css-only semantically-wise.

Reply:
I see your point. I think it's a preference of keeping markup dry vs. using CSS. As for the semantic question, I wonder if the pipe should be treated as a semantic item or rather a visual 'separator'. Purely semantically it's a valid list with list items. I usually opt for keeping the markup (HTML) dry and try to solve it with CSS. But up to your evaluation.

Reply:
Don't have a strong opinion either way. Separator is more versatile and can be added where needed without ` &:not(:last-of-type)::after` specification, which is brittle and is either everywhere the selector mathces or you'll have to add classes to specific sections you want to separate.

And css frameworks usually go for separators, e.g. MaterialUI has `<Divider/>

Reply:
Is the alternative to create a <Separator /> element we then conditionally (with a function?) add on the markup, except for the last item in a list?

Reply:
yes, map over children and add `<Separator/> for example
or add `<Separator/>` ourselves together with the children for better customizability

Reply:
Fine with me. From one side I find it overkill to create a specific component for it. On the other hand I fully understand your argument and specifically about a potential re-use of the component (to display the | separator in a styled way).

Reply:
i agree with @biocom here, the CSS is fine and is reusable. Adding a separator component to me is overkill

Reply:
Agree with David and Michel, this is what CSS is for :) 
let's try to be semantic with the mark up